iOS/New/Views/Manga/MangaView.swift

```diff
@@ -37,6 +53,15 @@ struct MangaView: View {
         self._viewModel = StateObject(wrappedValue: ViewModel(source: source, manga: manga))
         self.path = path
         self._scrollToChapterKey = State(initialValue: scrollToChapterKey)
+        let volumeMangas = UserDefaults.standard.array(forKey: "VolumeModeMangas") as? [String] ?? []
+        let chapterMangas = UserDefaults.standard.array(forKey: "ChapterModeMangas") as? [String] ?? []
+        if chapterMangas.contains(manga.key) {
+            self._displayMode = State(initialValue: .chapter)
+        } else if volumeMangas.contains(manga.key) {
+            self._displayMode = State(initialValue: .volume)
+        } else {
+            self._displayMode = State(initialValue: .default)
+        }
```

Comment:
i think instead of using a global array of keys, it would be better to do individual defaults like the reading mode. you can make the display mode enum use an Int for the rawValue, then use `Manga.chapterDisplayMode.key` for the default, like:
```swift
let key = manga.uniqueKey // use uniqueKey instead of key, which could be reused on different sources
let displayMode = MangaDisplayMode(rawValue: UserDefaults.standard.integer(forKey: "Manga.chapterDisplayMode.\(key)")) ?? .default
```

Reply:
Done.